core/src/main/java/io/spine/core/Events.java

```diff
@@ -218,6 +219,53 @@ public static boolean isExternal(EventContext context) {
     }
 
     /**
+     * Obtains the compacted version of the event, which should be used for storing.
```

Comment:
Let's discuss this description. Right now the name of the method does not correspond with it.

In particular, the description says that this is a procedure one would do before storing an event. And the rest of the content is about enrichments. So to me it looks more like a `clearEnrichments()` method. And even so I would adjust a few things.

However, we can stay with `compact()`. I like the name and the idea behind this name. However, there should be a more generic description, that this one.

Let's discuss this change.